src/app/dashboard/dashboard.module.ts

```diff
@@ -6,9 +6,20 @@ import { DashboardRoutingModule } from "./dashboard-routing.module";
 import { UnderNoticeState } from "./under-notice/state/under-notice.state";
 import { UnderNoticeComponent } from "./under-notice/under-notice.component";
 import { MaterialModule } from "../shared/material/material.module";
+import { TraineeSummaryComponent } from "./trainee-summary/trainee-summary.component";
+import { RevalidationHistoryComponent } from "./revalidation-history/revalidation-history.component";
+import { RevalidationNotesComponent } from "./revalidation-notes/revalidation-notes.component";
+import { RevalidationFormComponent } from "./revalidation-form/revalidation-form.component";
 
 @NgModule({
-  declarations: [DashboardComponent, UnderNoticeComponent],
+  declarations: [
+    DashboardComponent,
+    UnderNoticeComponent,
+    TraineeSummaryComponent,
+    RevalidationHistoryComponent,
```

Comment:
i think as this details screen is not a child of dashboard it would make sense to create a separate module alongside a routing file

Reply:
it is actually

Reply:
hmmm the revalidation history route seems to be completely independant to the dasbhboard no? no worries happy to leave it as it is